src/data/multiset/basic.lean

```diff
@@ -495,6 +495,10 @@ lemma strong_downward_induction_on_eq {p : multiset α → Sort*} (s : multiset
   s.strong_downward_induction_on H = H s (λ t ht h, t.strong_downward_induction_on H ht) :=
 by { dunfold strong_downward_induction_on, rw strong_downward_induction }
 
+/-- Another way of expressing `strong_induction_on`: the `(<)` relation is well-founded. -/
```

Comment:
Does this enable use with the equation compiler? If so, is it easy to construct an example for this docstring?

Reply:
Not directly: you would still need to write something along the lines of `using_well_founded { rel_tac := λ _ _, `[exact ⟨_, well_founded_lt⟩] }`. And if you want to do that, why not use `strong_induction_on` directly? Or we can bundle `well_founded_lt` up into a `has_well_founded` instance, but there are multiple well-founded less-than relations on multisets, and they can be incomparable (e.g. "`s` is a strict subset of `t`" and `card s < card t`) so that sounds like the wrong thing to do.

Reply:
Oh, I totally misread this as `has_well_founded`.